runners/spark/src/main/java/org/apache/beam/runners/spark/util/BroadcastHelper.java

```diff
@@ -39,48 +38,17 @@
    * in View objects. By default this property is not set, and values are coded using
    * the appropriate {@link Coder}.
    */
-  public static final String DIRECT_BROADCAST = "beam.spark.directBroadcast";
-
   private static final Logger LOG = LoggerFactory.getLogger(BroadcastHelper.class);
 
-  public static <T> BroadcastHelper<T> create(T value, Coder<T> coder) {
-    if (Boolean.parseBoolean(System.getProperty(DIRECT_BROADCAST, "false"))) {
-      return new DirectBroadcastHelper<>(value);
-    }
-    return new CodedBroadcastHelper<>(value, coder);
+  public static <T> BroadcastHelper<T> create(byte[] bytes, Coder<T> coder) {
```

Comment:
Can get rid of the abstraction.
I don't see a reason to use "direct" translation since Beam heavily relies on Coders.